apis/overview/lifecycle.md

```diff
@@ -27,33 +27,62 @@ The following changes are backward compatible:
+
+You may use APIs in Tech Preview for free while they are in preview. Do not use these APIs in Production. They are available for testing only, to gather feedback before releasing for General Availability.
+
+**Note**: We reserve the right to remove items in Tech Preview from the ecosystem at any time. In which case, we will no longer support them.
+
+### General Availability
+
+APIs in General Availability are available for production ready use. We provide full support for these APIs. APIs can remain in General Availability for an unlimited period however, substantial changes to the structure and function of the API can require a change in version. In these cases, when we release the new API version to GA, we transition the older version to a Deprecated status and you are encouraged to move to the new version.
+
+### Deprecated
+
+We deprecate an API when we replace it by a different API or update it to a new version. If you are using the deprecated API for your solutions, you are encouraged to update to the new API to take advantage of the new capabilities. We provide limited support to the deprecated API however, we fully support upgrading to a new version during this time. This policy excludes new development on a deprecated API. New customers must use the currently released version.
+
+We keep deprecated APIs available and supported for a maximum of one year.
+
+### End Of Support
+
+When an API is deprecated, we keep it available for one year, allowing time for you to upgrade. After this time, we consider it at End of Support and remove it from the iTwin Platform ecosystem.
+
+## Support Services
+
+| **Technical Preview** | **General Availability**         | **Deprecated**                             | **End Of Support**                |
+| --------------------- | -------------------------------- | ------------------------------------------ | --------------------------------- |
+| **Limited Support**   | **Full Support**                 | **Limited Support**                        | **Upgrade Only**                  |
+| GitHub community      | Bentley support                  | Bentley support (for one year)             | Upgrade to the latest API version |
+| Stack Overflow        | GitHub community                 | Github community                           |                                   |
+| Security updates      | Stack Overflow                   | Security updates                           |                                   |
+| New functionality     | Security updates                 | Maintenance updates (Bentley's discretion) |                                   |
+| Bug fixes             | Maintenance updates              | No enhancements or new functionality       |                                   |
+|                       | New functionality - Non breaking |                                            |                                   |
+
+<!-- ## Deprecation Guidelines
```

Comment:
why this part is commented out?

Reply:
Michelle said to leave it out for now. But we might use it in a future revision

Reply:
Should we still keep it as commented out in public repo?

Reply:
Removed it from this branch. Moved to a [separate branch](https://github.com/iTwin/platform-api-documentation/blob/api-lifecycle-future-update/apis/overview/lifecycle.md?plain=1#L72) for future update.